RadixWallet/Features/FactorSourceAccess/FactorSourcePerformer.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+// MARK: - FactorSourcePerformer
+/// An enum containing the necessary information to use a given `FactorSource`.
+///
+/// This becomes handy to unify factor sources that can be used directly (`device`, `ledger` & `arculusCard`) and those
+/// that require user input to retrieve the associated `MnemonicWithPassphrase` (`offDeviceMnemonic` & `password`).
+enum FactorSourcePerformer: Sendable, Hashable {
```

Comment:
Hmm I dont like this name. Can we come up with something more clear. And it is actually somewhat related to `PrivateHierarchicalDeterministicFactorSource` in Sargon. 

`Private` means it holds secrets. Which this does.  `PrivateFactorSource` seems like a better name.